### PR TITLE
Removed edit mode and renamed the existing raw edit mode to edit mode

### LINKED ISSUE #1325: Remove/deprecate edit mode and call raw edit mode 'edit'.
Remember to map the shortcut Shift-E to Shift-F.

### PR DESCRIPTION
**Pull request by percyliang**
Fixes #1325 
@kashizui

### CODE REVIEW

File: codalab/apps/web/static/js/worksheet/worksheet_interface.js [hunk 1]
```diff
@@ -120,65 +108,45 @@ var Worksheet = React.createClass({
...
-    toggleRawMode: function(val){
-        if(typeof(val)=='undefined'){
-            if(this.state.rawMode){
-                ///TODO grab val the react way
-                ws_obj.state.raw = $("#raw-textarea").val().split('\n');
-                if(this.canEdit()){
-                    this.saveAndUpdateWorksheet(true);
-                }
+    toggleEditMode: function(editMode) {
+        if (typeof(editMode) == 'undefined')
```
Comment (kashizui) [inline codalab/apps/web/static/js/worksheet/worksheet_interface.js]: nit: I know this was not added in this diff, and not sure what our style guidelines are (maybe we can adopt the Google JavaScript Style guide somewhere in the wiki), but I think these comparisons should usually be `===`.

File: codalab/apps/web/static/js/worksheet/worksheet_items.js [hunk 2]
```diff
@@ -1,5 +1,10 @@
 /** @jsx React.DOM */
 
+/*
+All of the editing functionality here is deprecated, since it's too complicated to maintain
+and to keep in sync the markdown.  We are switching to pure markdown editing.
```
Comment (kashizui) [inline codalab/apps/web/static/js/worksheet/worksheet_items.js]: "with the markdown"

File: codalab/apps/web/templates/web/worksheets/detail.html [hunk 3]
```diff
@@ -38,79 +38,39 @@
...
-                        <td><kbd>c</kbd> Jump to the console</td>
+                      <td><kbd>c</kbd></td><td>Focus on web terminal</td>
                     </tr>
                     <tr>
-                        <td><kbd>E</kbd> Toggle editing mode</td>
+                      <td><kbd>ESC</kbd></td><td>Focus on worksheet</td>
                     </tr>
                     <tr>
-                        <td><kbd>F</kbd> Raw editing mode for worksheet</td>
+                      <td><kbd>C</kbd></td><td>Show/hide web terminal</td>
```
Comment (kashizui) [inline codalab/apps/web/templates/web/worksheets/detail.html]: What is the semantics behind capitalizing some of these keys and not others?

**Comment by kashizui** [comment]
Overall LGTM!

File: codalab/apps/web/templates/web/worksheets/detail.html (hunk 3 above)
Comment (percyliang) [inline codalab/apps/web/templates/web/worksheets/detail.html]: C is shift-c (changed to make this clear)

**Pull request merged**